DependencyInjection/HttplugExtension.php

```diff
@@ -56,6 +61,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_journal_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.plugin.history');
```

Comment:
ah, read the doc now. maybe throw an InvalidConfigurationException if its explicitly specified? then again, why not allow users to activate the history plugin and do their own thing, even in prod mode if they feel its a good idea?